Asgardia.Constitution.Proposal.txt

```diff
@@ -42,51 +42,50 @@ of our new space nation created by us;
 
 Article 3. Mission
 To ensure the existence and development of Asgardia and the new humanity in space.
 
 Article 4. Supreme Values of Asgardia
 § 1. Asgardia’s Supreme Values are the most common and permanent foundations of the self-identification of Asgardian citizens which form the commonality of Asgardian society and lie at the heart of the objectives and activities of Asgardian state bodies, citizens, and associations.
 § 2. The Constitution, laws, and other regulations of Asgardia, as well as the activities of its state bodies, implement and protect Asgardia’s Supreme Values.
-§ 3. Asgardia’s absolute supreme value is humanity striving into the limitless future of the limitless Universe, an unstoppable quest to understand and transform the Universe.
+§ 3. Asgardia’s absolute Supreme Value is humanity striving into the limitless future of the limitless Universe, an unstoppable quest to understand and transform the Universe.
 § 4. Asgardia’s Supreme Values in fulfilling its Mission are:
- a. peace in Space and the peaceful settlement of the Universe;
+ a. peace in space and the peaceful settlement of the Universe;
  b. protection of all of humanity from space-originating threats;
- c. ensuring equality of opportunity in Space for all Asgardian citizens;
- d. unity of humanity as a community in Space;
+ c. ensuring equality of opportunity in space for all Asgardian citizens;
+ d. unity of humanity as a community in space;
  e. human dignity, human rights, freedoms, and the harmonious development of the individual;
- f. human life, love, children, and family, and the propagation of the human species;
+ f. human life, love, children, family, and the propagation of the human species;
  g. supremacy of law; 
  h. space ecology;
  i. community and mutual support; 
- j. scientific, artistic creativity, and the belief in the boundless potential of human reason, knowledge, labour and progress;
- k. peace, tranquillity, security and confidence;
- l. morality, fairness and freedom;
- m. harmonious existence of the individual, society, and the state.
-§ 5. Asgardia’s Supreme Values are equally important and equally implemented in state administration. In the event of a conflict between the Supreme Values, the absolute supreme value of Asgardia prevails.
+ j. scientific advancement, artistic creativity, and the belief in the boundless potential of human reason, knowledge, labor, and progress;
+ k. peace, tranquility, security, and confidence;
+ l. morality, fairness, and freedom;
+ m. harmonious existence of the individual, society, and the Space Nation.
+§ 5. Asgardia’s Supreme Values are equally important and equally implemented in state administration. In the event of a conflict between the Supreme Values, the absolute Supreme Value of Asgardia prevails.

```

Comment:
Are we trying to say a conflict between individual Supreme Values, or are we trying to say between Supreme Values and future laws? I believe the latter, but we need to rephrase if this is the case